CONTRIBUTING.md

```
@@ -0,0 +1,163 @@
+
+## 목차
+
+- [브랜치 전략](#-브랜치-전략)
+- [이슈 작성](#-이슈-작성)
+- [커밋 컨벤션](#-커밋-컨벤션)
+- [PR 규칙](#-pr-규칙)
+- [코드 컨벤션](#-코드-컨벤션)
+
+## 🌿 브랜치 전략
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)